resources/views/articles/index.blade.php

```diff
@@ -9,7 +9,7 @@
     <div class="row border-bottom py-3">
         <div class="col-lg-6 order-2 order-lg-1">
             <p class="mb-4">{{ $article->published_at->toDateString() }}</p>
-            <h6 class="mb-4 font-weight-bold">{{ $article->title }}</h6>
+            <a href="{{ route('about.articles.show', $article) }}"><h6 class="mb-4 font-weight-bold">{{ $article->title }}</h6></a>
```

Comment:
```suggestion
           <h6 class="mb-4 font-weight-bold"><a href="{{ route('about.articles.show', $article) }}" class="link-no-underline">{{ $article->title }}</a></h6>
```

it behaves better when link (inline) is inside header (block) https://stackoverflow.com/questions/7023512/which-is-more-correct-h1a-a-h1-or-ah1-h1-a/7023551#7023551

Reply:
Fixed. Thanks